php/libraries/Database.class.inc

```diff
@@ -11,6 +11,9 @@
  * @link     https://www.github.com/aces/Loris-Trunk/
  */
 define("DEBUG", false);
+define('ESCAPE_ALL', 0);
+define('ESCAPE_NOQUOTES', 1);
+define('ESCAPE_NONE', 2);
```

Comment:
I don't see why you include both define's nd consts below. The const should be enough.